### PR TITLE
lodash-fp support

### PR DESCRIPTION
**Pull request by megawac**

### CODE REVIEW

File: test/fixtures/lodash-fp-direct/expected.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  'add': _lodashMathAdd2['default'],
+  'take': _lodashArrayTake2['default'],
+  'reject': _lodashCollectionReject2['default']
+});
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }
+
+var mapper = _lodashFp.map(_lodashFp.add(1));
+var result = mapper([1, 2, 3]);
+_lodashFp.take(1, _lodashFp.reject(Boolean, result));
```
Comment (megawac) [inline test/fixtures/lodash-fp-direct/expected.js]: If you get a second @jdalton mind sanity checking this output. It seems correct to me.

Comment (jdalton) [inline test/fixtures/lodash-fp-direct/expected.js]: The fp part? It looks good.

File: test/fixtures/lodash-fp-direct/expected.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+var _lodashCollectionReject = require('lodash/collection/reject');
+
+var _lodashCollectionReject2 = _interopRequireDefault(_lodashCollectionReject);
+
+var _lodashFpConvert = require('lodash-fp/convert');
+
+var _lodashFpConvert2 = _interopRequireDefault(_lodashFpConvert);
+
+var _lodashFp = (0, _lodashFpConvert2['default'])({
```
Comment (jdalton) [inline test/fixtures/lodash-fp-direct/expected.js]: Why the `0` in `(0,  _lodashFpConvert2['default'])` ?

File: test/fixtures/lodash-fp-direct/expected.js (hunk 1 above)
Comment (jdalton) [inline test/fixtures/lodash-fp-direct/expected.js]: This is so crazy cool.

File: test/fixtures/lodash-fp-direct/expected.js (hunk 2 above)
Comment (megawac) [inline test/fixtures/lodash-fp-direct/expected.js]: It's something `babel` adds, I'm not entirely sure why

https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&code=import%20x%20from%20%27x%27%3B%0A%0Avar%20y%20%3D%20x%28{}%29

Comment (jdalton) [inline test/fixtures/lodash-fp-direct/expected.js]: maybe in case it's an `eval` so it would be an indirect eval?

**Comment by jadbox** [comment]
Awesome!

**Pull request merged**